account_credit_control/models/credit_control_run.py

```diff
@@ -140,8 +140,7 @@ def _generate_credit_lines(self):
             else:
                 report += _(
                     "Policy \"<b>%s</b>\" has not generated any "
-                    "Credit Control Lines.<br/>" % policy.name
-                )
+                    "Credit Control Lines.<br/>") % policy.name
```

Comment:
I didn't notice this mistake in translations. Can you redo translations for not requiring to re-translate this?

Reply:
Isn't it automatically fixed by @oca-travis ?

Reply:
I don't think so as the msgid is changing. Can you confirm if so?

Reply:
Well, the only way to proof or disproof my claim is just merging it and wait, and if it is not fixed automatically, then open new PR to fix that. My basis is in https://github.com/OCA/knowledge/commit/2c04cf8566f564b60881839015cd1ab2b6fc13dd, it's a automatically generated commit fixing the `.pot` file

Reply:
Also you can clearly see in https://github.com/OCA/account-financial-tools/blob/11.0/account_credit_control/i18n/account_credit_control.pot#L1056 (and the other `.po` files) that there is nothing to fix :S

Reply:
OK, that's the kind of check I wanted you to do. We can merge then. Thanks!